content/src/content/jcr_root/apps/core/wcm/components/tabs/v1/tabs/README.md

```diff
@@ -33,7 +33,7 @@ The component policy dialog allows definition of allowed components for the Tabs
 ### Edit Dialog Properties
 The following properties are written to JCR for this Tabs component and are expected to be available as `Resource` properties:
 
-1. `./activeItem` - defines the item that is active by default.
+1. `./activeItem` - defines the item that is active by default (the name of the active item `Resource`).
```

Comment:
I would phrase it as follows: 
defines the name of the item that is active by default